aws-emrserverless-application/docs/workerconfiguration.md

```diff
@@ -72,3 +74,15 @@ _Pattern_: <code>^[1-9][0-9]*(\s)?(GB|gb|gB|Gb)$</code>
 
 _Update requires_: [No interruption](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/using-cfn-updating-stacks-update-behaviors.html#update-no-interrupt)
 
+#### DiskType
+
+Per worker DiskType resource. Shuffle optimized and Standard are only supported types and specifying diskType is optional
+
+_Required_: No
+
+_Type_: String
+
+_Pattern_: <code>^(SHUFFLE_OPTIMIZED|[Ss]huffle_[Oo]ptimized|STANDARD|[Ss]tandard)$</code>
+
+_Update requires_: [No interruption](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/using-cfn-updating-stacks-update-behaviors.html#update-no-interrupt)
```

Comment:
I am not sure about this. If an application is started, update will fail and require stopping the application to perform an update. Does this use case still meet "No interruption" definition?

Reply:
This docs are auto-generated. It probably assumes that update is only possible when application is stopped. And in that case, there will be no interruptions.